_posts/2017-04-05-airpods-bluetooth-interference-mac.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title:  "Troubleshoot: Poor AirPod Connection with Mac Mini"
+date:   2017-04-05 09:22:07 -0400
+categories: jekyll tech ccannon Apple AirPods MacMini
+---
+I recently received a pair of AirPods as a birthday present and I must say, they are incredible. However, I experienced some difficulty right away with connecting them to my Mac Mini. They would pair, but sound would come over broken, jumping from one ear to the other, with nothing clear coming through. I decided to contact Apple Support after trying the most basic troubleshooting I could think of: unpairing and repairing the AirPods from my AppleID, and pairing the AirPods directly to the Mac Mini.
+
+### The Solution
+Basically, after 4 different case numbers, we determined that the most likely cause of my issue was an abundance of Bluetooth interference from my previous generation Magic Keyboard, Magic Trackpad, and Magic Mouse. The final Apple Support advisor I spoke with suggested that I power off my keyboard and trackpad, using only the mouse to pair the AirPods. Once I did, music came in consistently clear like it hadn't before.
+
+### The Troubleshooting Experience
+I must give credit to Apple for their online support experience. Since I purchased my first Mac in 2013, the experience has simplified and become far more accessible. Save for one particular advisor (who advised me to turn my Mac off and on again, after I provided him my two previous case numbers), everyone I interacted with was polite and professional and, most importantly to me, provided me with the next most logical step to work through my problem. Finally, I called in to Apple to speak with a senior advisor, all previous interactions had been over chat. Seeing that we had exhausted the standard troubleshooting checklist, he decided to "think outside the box".
```

Comment:
Personally, I would give like a sentence about how the support service has become simplified. You have to think, you may get people who are not big techies or younger readers or something idk. I think it would help, but it is up to you.